frontend/src/container/WorkspaceContent.jsx

```diff
@@ -723,15 +724,22 @@ export class WorkspaceContent extends React.Component {
               currentSpace={props.currentWorkspace}
               breadcrumbs={breadcrumbs}
             />
-
             <PageContent parentClass='workspace__content'>
-              {userRoleIdInWorkspace >= ROLE.contributor.id && (
-                <DropdownCreateButton
-                  folderId={null} // null because it is workspace root content
-                  onClickCreateContent={this.handleClickCreateContent}
-                  availableApp={createContentAvailableApp}
+              <div className='buttons'>
```

Comment:
The class names are usually a composition with the parent's name, so instead of 'buttons' I would prefer 'workspace__content__buttons'